web-ui/src/backup_account/backup_email/backup_email.js

```diff
@@ -54,8 +54,14 @@ export class BackupEmail extends React.Component {
       body: JSON.stringify({
         csrftoken: [browser.getCookie('XSRF-TOKEN')]
       })
-    }).then(() => this.props.onSubmit('success'));
-  }
+    }).then((response) => {
+      if (response.status === 204) {
+        this.props.onSubmit('success');
+      } else {
+        this.props.onSubmit('error');
+      }
+    });
```

Comment:
Right... I suspected that was true. Can we use `response.ok` then, to check for a successful response?